ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl/Pure.hs

```diff
@@ -320,12 +319,6 @@ implSnapshotFromIS is = MempoolSnapshot {
   implSnapshotGetTxsAfter IS{isTxs} =
     TxSeq.toTuples . snd . TxSeq.splitAfterTicketNo isTxs
 
-  implSnapshotGetTxsForSize :: InternalState blk
```

Comment:
Nix was painfully slow to load the webpage, but the Nix build failure is because this was the only usage of `Word32`. The corresponding import is now unused.